.zenodo.json

```diff
@@ -0,0 +1,130 @@
+{
+    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "license": "BSD-2-Clause",
+    "title": "SunPy",
+    "upload_type": "software",
+    "creators": [
+        {
+            "affiliation": "1 - DKIST. 2 - Aperio Software",
+            "name": "Stuart Mumford",
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
+        },
+        {
+            "affiliation": "Aperio Software",
```

Comment:
What about?
```suggestion
            "affiliation": "Independent",
```

Reply:
Actually if this for 1.0, University of Leicester should be ok.